inlang/source-code/sdk2/src/project/loadProjectFromDirectory.ts

```diff
@@ -5,6 +5,9 @@ import { uuidv4, type Lix } from "@lix-js/sdk";
 import type fs from "node:fs/promises";
 // eslint-disable-next-line no-restricted-imports
 import nodePath from "node:path";
+import type { InlangPlugin } from "../plugin/schema.js";
+import { insertNestedBundle } from "./util/insertBundleNested.js";
+import { fromMessageV1 } from "./util/fromMessageV1.js";
 
 /**
  * Loads a project from a directory.
```

Comment:
(Not very important) Rename `loadProjectFromDirectory` to `loadProjectFromDirectoryInMemory` to clarify that this function loads a project in memory? Later we can add `loadProjectFromDirectoryWithTempFile()` or similar functions. Yes, long name but it's only called once.